etl/steps/data/grapher/geography/2024-07-18/latitude.py

```diff
@@ -0,0 +1,57 @@
+"""Load a garden dataset and create a grapher dataset."""
+
+from etl.helpers import PathFinder, create_dataset
+
+# Get paths and naming conventions for current step.
+paths = PathFinder(__file__)
+
+
+def run(dest_dir: str) -> None:
+    #
+    # Load inputs.
+    #
+    # Load garden dataset.
+    ds_garden = paths.load_dataset("nat_earth_110")
+
+    # Read table from garden dataset.
+    tb = ds_garden["nat_earth_110"].reset_index()
+    tb = tb[["name", "latitude_centroid", "absolute_latitude"]]
+    tb["year"] = 2024
```

Comment:
I think it might be useful to have these not just for 2024 but maybe dating back for every year back to 1900s?

Reply:
Great point! I'll add it for 1900-2100.